schema/time/index.js

```diff
@@ -0,0 +1,33 @@
+import { GraphQLString, GraphQLBoolean, GraphQLInt, GraphQLObjectType } from "graphql"
+
+export const timeFields = () => {
+  return {
+    day: { type: GraphQLInt },
+    wday: { type: GraphQLInt },
+    month: { type: GraphQLInt },
+    year: { type: GraphQLInt },
+    hour: { type: GraphQLInt },
+    min: { type: GraphQLInt },
+    sec: { type: GraphQLInt },
+    dst: { type: GraphQLBoolean },
+    unix: { type: GraphQLInt },
+    utc_offset: { type: GraphQLInt },
+    zone: { type: GraphQLString },
+    iso8601: { type: GraphQLString },
+  }
+}
+
+export const TimeType = new GraphQLObjectType({
+  name: "Time",
```

Comment:
I think the name is too broad and ambiguous. I would expect a ‘Time’ type to be able to represent any sort of time, but this type (the resolver at least) is specifically about Artsy’s system time.

Reply:
Good point! What do you think about embedding it with `system`? So the query would look like:

```
{
  system {
    time { ... }
  }
}
```

(In addition t renaming `TimeType` to `SystemTimeType`?)

Reply:
This makes sense to me